spring-cloud-gcp-docs/src/main/asciidoc/spanner.adoc

```diff
@@ -257,6 +257,67 @@ Column naming is generated by the `PropertyNameFieldNamingStrategy` by default d
 The `@Column` annotation optionally provides a different column name than that of the property.
 
 
+==== Embedded Objects
+
+If an object of type `B` is embedded as a property of `A`, then the columns of `B` will be saved in
+the same Cloud Spanner table as those of `A`. If `B` has primary key columns, those columns will be
+included in the primary key of `A`. `B` can also have embedded properties. Embedding allows reuse
+of columns between multiple entities, and can be useful for implementing parent-child situations, because
+Cloud Spanner requires child tables to include the key columns of their parents.
+
+For example:
+
+[source,java]
+----
+class X {
+  @PrimaryKey
+  String grandParentId;
+
+  long age;
+}
+
+class A {
+  @PrimaryKey
+  @Embedded
+  X grandParent;
+
+  @PrimaryKey(keyOrder = 2)
+  String parentId;
+
+  String value;
+}
+
+@Table(name = "items")
+class B {
+  @PrimaryKey
+  @Embedded
+  A parent;
+
+  @PrimaryKey(keyOrder = 2)
+  String id;
+
+  @Column(name = "child_value")
+  String value;
+}
+----
+
+Entities of `B` can be stored in a table defined as:
+
+[source,sql]
+----
+CREATE TABLE items (
+    grandParentId STRING(MAX),
+    parentId STRING(MAX),
+    id STRING(MAX),
+    value STRING(MAX),
+    child_value STRING(MAX),
+    age INT64
+) PRIMARY KEY (grandParentId, parentId, id)
+----
+
+Note that embedded properties' column names must all be unique.
+
+
```

Comment:
Please also improve spanner-sample and AbstractSpannerIntegrationTest to include embedded properties.

Reply:
We actively did not choose to add complex functionality to the samples to not overwhelm somebody checking this out, so I don't think we want to include it in there. 

As for the integration test, it is already tested in integration tests. Take a look at the Trade.java entity type we use for the integration tests, and it has an embedded property.